spark-cassandra-connector/src/main/scala/com/datastax/spark/connector/mapper/JavaBeanColumnMapper.scala

```diff
@@ -28,6 +34,24 @@ class JavaBeanColumnMapper[T : ClassTag](columnNameOverride: Map[String, String]
     method.getParameterTypes.size == 1 &&
     method.getReturnType == Void.TYPE
 
+  private def annotationForFieldName(fieldName: String): String = {
```

Comment:
That said I'm fine if you just want to change the return of this function to Option[String] :) That's the only thin I think is important here